haystack/dataclasses/chat_message.py

```diff
@@ -14,6 +15,14 @@ class ChatRole(str, Enum):
     USER = "user"
     SYSTEM = "system"
     FUNCTION = "function"
+    TOOL = "tool"
+
+
+@dataclass
+class ToolCall:
+    id: str
+    tool_name: str
+    arguments: Dict[str, Any]
```

Comment:
These changes to `ChatMessage` should be in haystack or haystack-experimental?

Reply:
Everything goes into experimental until we stabilize the API.